floorplan-cli/build.gradle

```
@@ -17,25 +17,26 @@ application {
     getMainClass().set("com.zynger.floorplan.FloorPlanCliKt")
 }
 
-tasks.register('embarkVersionFile') {
+def embarkVersionFile = tasks.register('embarkVersionFile') {
     inputs.property("floorPlanVersion", floorPlanVersion)
 
-    outputs.file(
-        Paths.get(sourceSets.main.output.resourcesDir.path)
-            .resolve("floorplan-cli-version.properties")
-    ).withPropertyName("floorplanVersionPropertiesFile")
+    def genDir = layout.buildDirectory.dir('generated-props')
+    outputs.dir(genDir).withPropertyName("floorplanVersionGeneratedPropertiesFile")
```

```suggestion
    outputs.dir(genDir)
```
the name of the output isn't used anywhere so IMO it is redundant.